core/adapter/reslice.h

```diff
@@ -183,10 +186,37 @@ namespace MR
         default_type from[3], inc[3];
         default_type norm;
         const transform_type transform_, direct_transform;
+
+        void normalise (default_type&) const;
```

Comment:
Yeah, the anonymous namespace trick should do it. Probably why SFINAE has been done like that in the past.